### ISSUE TITLE
Epsilon scheduler

### ISSUE THREAD

**Issue by MUCDK** [theislab/moscot#191]
Currently, ```epsilon``` is assumed to be of type ```float```.

I think we should also allow schedulers, see

https://ott-jax.readthedocs.io/en/latest/_autosummary/ott.geometry.epsilon_scheduler.Epsilon.html
